Formula/k/kubevpn.rb

```diff
@@ -0,0 +1,25 @@
+class Kubevpn < Formula
+  desc "Offers a Cloud-Native Dev Environment that connects to your K8s cluster network"
+  homepage "https://www.kubevpn.cn"
+  url "https://github.com/kubenetworks/kubevpn.git",
+      tag:      "v2.2.11",
+      revision: "98d88ac542c06e8f66937e7f57b5ceae514b0c30"
+  license "MIT"
+  head "https://github.com/kubenetworks/kubevpn.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["OUTPUT_DIR"] = bin
+    ENV["VERSION"] = version
+    ENV["GIT_COMMIT"] = Utils.git_head
+    ENV["BRANCH"] = Utils.git_branch
+    system "make", "kubevpn"
+  end
+
+  test do
+    system "#{bin}/kubevpn", "version"
+    assert_predicate testpath/".kubevpn/config.yaml", :exist?
+    assert_predicate testpath/".kubevpn/daemon", :exist?
```

Comment:
Are the files under `.kubevpn` created during the `kubevpn version` invocation?

Reply:
Yes, this is init logic, kubevpn use ~/.kubevpn to store config info. use ~/.kubevpn/daemon to   store socks file and pid file

Reply:
Every command will to check `.kubevpn` exist or not(include command version), if not exist, it will try to create it, if create failed, command will panic exit.